mavros/launch/px4_config.yaml

```diff
@@ -44,20 +44,20 @@ cmd:
 
 # global_position
 global_position:
-  frame_id: "map"             # origin frame
-  child_frame_id: "base_link" # body-fixed frame
+  frame_id: "local_origin"  # origin frame
+  child_frame_id: "fcu_utm" # body-fixed frame
```

Comment:
Why `fcu_utm`?

Reply:
can not remember any more. Any suggestions? it can not be fcu, since this would break the tree.

Reply:
`fcu_global`